modules/rosa-sts-aws-requirements.adoc

```diff
@@ -143,4 +143,4 @@ If you do not specify a profile, the default AWS profile is used.
 == Security requirements
 * Volume snapshots will remain within the customer's AWS account and customer-specified region.
 * Red Hat must have ingress access to EC2 hosts and the API server from allow-listed IP addresses.
-* Red Hat must have egress allowed to the documented domains. See the "AWS firewall prerequisites" section for the designated domains.
+* Red Hat must have egress allowed to the documented domains. See the AWS firewall prerequisites section for the designated domains.
```

Comment:
I see that you removed the quotes from this xref, but given that it is a module, wouldn't you either (1) want to keep them; or (2) insert an `Additional resources` entry after this module to link to the content? I ask because the preview you provided about removing unnecessary quotes does not line up with this change.